faq.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>FAQ | 판다마켓</title>
+<!-- pretendard web-font: https://github.com/orioncactus/pretendard -->
+<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+<link rel="stylesheet" href="style.css">
+<link rel="icon" href="favicon.png" type="image/png">
+</head>
+
+<body>
+<div class="wrap">
+  <header id="header">
+    <div class="container header">
+      <h1 class="site-name">
+        <a href="/" class="root-link">
+          <img src="img/logo.png" alt="로고 이미지" class="logo">
+          <img src="img/site-name.png" alt="판다마켓">
+        </a>
+      </h1>
+      <a href="/login.html" class="btn">로그인</a>
+    </div>
+    <!-- /.container -->
+  </header>
+
+  <section class="section">
+    <div class="container">
+      <h2>FAQ</h2>
+    </div>
+    <!-- /.container -->
+  </section>
+  <!-- /.section -->
+
+  <div class="bottom-visual bg-sky">
+    <div class="container">
+      <h2 class="heading">믿을 수 있는<br>판다마켓 중고 거래</h2>
+    </div>
+    <!-- /.container -->
+  </div>
+  <!-- / bottom-visual -->
+
+  <footer id="footer">
+    <div class="container">
+      <p class="cr">&copy;codeit - 2024</p>
+      <ul class="links">
+        <li><a href="privacy.html">Privacy Policy</a></li>
+        <li><a href="faq.html">FAQ</a></li>
+      </ul>
```

Comment:
```suggestion
    <nav aria-label="footer navigation">
      <ul class="links">
        <li><a href="privacy.html">Privacy Policy</a></li>
        <li><a href="faq.html">FAQ</a></li>
      </ul>
    <nav>
```

지금도 충분히 시멘틱하게 잘 해주셨지만, 네비게이션의 경우 메뉴를 `nav` 태그와 접근성을 위한 `aria-label` 까지 사용해주면 더 좋겠습니다. 웹표준을 따라 작성하면 문서의 구조와 의미를 보다 더 정확하게 검색엔진과 스크린리더에 전달할 수 있으니까요. 😄